quartz/i18n/locales/ru-RU.ts

```diff
@@ -62,7 +62,12 @@ export default {
     rss: {
       recentNotes: "Недавние заметки",
       lastFewNotes: ({ count }) =>
-        `Последн${getForm(count, "яя", "ие", "ие")} ${count} замет${getForm(count, "ка", "ки", "ок")}`,
+        `Последн${getForm(count, "яя", "ие", "ие")} ${count} замет${getForm(
+          count,
+          "ка",
+          "ки",
+          "ок",
+        )}`,
```

Comment:
hmm this seems like it is not related to this PR?

Reply:
It's definitely not but `npm run format` changed it when I ran the command. Should it be left in?

Reply:
let's remove this diff then